logparser/devices/inputdevices.py

```diff
@@ -118,7 +118,11 @@ class InputFileDevice(InputDevice):
     def __init__(self, file_path, state):
         """Initialize the device with the specified file path."""
         super(InputFileDevice, self).__init__(state)
-        self.stream = open(file_path, "r")
+        try:
+            self.stream = open(file_path, "r")
+        except IOError:
+            print("The file cannot be opened. Verify the file exists and you can access it")
+            exit(-1)
```

Comment:
We shouldn't call `exit` inside the library. It's not expected that an application using the logparser library suddenly quit because of it. The application would expect to receive an exception.